### ISSUE TITLE
Error messages related to missing structural members should have priority over those related to accessibility

### ISSUE THREAD

**Issue by mor-n4** [eclipse/n4js#785]
C.n4js:
```javascript
export public class C {
	private field: int;
	private set acc(value: int) {}
}
```
D.n4js
```javascript
import {C} from "C";


let c: ~i~C;
// XPECT errors --> "field field is not available for structurally referenced type ~i~C." at "field"
c.field; // current error is "The field field is not visible."
// XPECT errors --> "setter acc is not available for structurally referenced type ~i~C." at "acc"
c.acc = 0; // current error is "The setter acc is not visible."
```
